pages/about.tsx

```diff
@@ -79,22 +82,29 @@ export default function About({ loginContext }: AboutProps) {
         <meta name="description" content="About" />
       </Head>
       <Navbar
+        showNotification={showNotification}
         fill="black"
-        className={clsx('bg-iflightorange', 'text-black')}
+        className={clsx('bg-ifbeige', 'text-black')}
         loginContext={loginContext}
       />
       <main
         className={clsx(
-          'bg-iflightorange',
+          'bg-ifbeige',
           'flex-1',
           'items-center',
           'flex',
           'flex-col'
         )}
       >
         <PageBanner
-          title="About the Incentivized Testnet"
-          text="Sign up for the Iron Fish incentivized testnet to help make Iron Fish great 💖. Participate to earn testnet points (see Testnet Guidelines below for more details)."
+          title={
+            <>
+              Incentivized Testnet
+              <br />
+              Welcome to Phase 2.
+            </>
+          }
+          text="Get ready to claim your share of 315,000 $IRON! To make Phase 2 fair, we’ve removed weekly limits on point earnings. We’re also introducing a new leaderboard; but don’t worry, all accumulated Phase 1 points will still count to your overall score. You can earn $IRON in a few different ways:"
```

Comment:
I think somewhere else this "315,000" number was computed from the phase + pool data, should we do that here too?